app/models/bulkrax/csv_entry.rb

```diff
@@ -76,7 +76,7 @@ def build_export_metadata
       self.parsed_metadata = {}
       self.parsed_metadata['id'] = hyrax_record.id
       self.parsed_metadata[source_identifier] = hyrax_record.send(work_identifier)
-      self.parsed_metadata['model'] = hyrax_record.has_model.first
+      self.parsed_metadata[mapping['model'] & ['from']&.first || 'model'] = hyrax_record.has_model.first
```

Comment:
without the spaces around the ampersand I got this rubocop error: "Surrounding space missing for operator &" 🤷🏾‍♀️